conf/ocs_version/ocs-4.5.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 DEPLOYMENT:
-  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-olm-operator:latest-4.5"
+  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-bundle-operator:latest-4.5"
```

Comment:
do we need this for 4.4 as well?

Reply:
No, 4.4 does not support bundles.

Reply:
btw: OCP 4.4 does include support for bundles but we (RH) don't support it.